util/stmtsummary/evicted_test.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/pingcap/tidb/types"
 )
 
+var _ = Suite(&testStmtSummarySuite{})
```

Comment:
pollutions?

Reply:
i guess put `statement_summary_test.go` and `evicted_test.go` in one pr is too large, so i migrate `evicted_test.go` in another pr #26871 .
so in this pr, `evicted_test.go` is still use gocheck,  and i added  some codes to make unit test pass

Reply:
these seems ugly, maybe just put them in one pr is fine?
i will review this part and what is your advise?

Reply:
I get the point. You can write as this. The next time you may elaborate a bit in the description about essential changes so that reviewer can understand why you do something non-trivial.

Reply:
ok, get it